### PR TITLE
refactor(cli): use object args for cli commands

### LINKED ISSUE #1035: use interfaces for cli commands arguments
The cli commands have a large number of arguments, which can lead to confusion and potential mistakes. Using an object with an appropriate interface would make it easier to use and reduce risk of errors

### PR DESCRIPTION
**Pull request by ctrlc03**
# Description

Cleanup cli commands by using object args, as well as amend tests to reflect this change

## Related issue(s)

fix #1035 

## Confirmation

- [x] I have read and understand MACI's [contributor guidelines](https://github.com/privacy-scaling-explorations/maci/blob/dev/CONTRIBUTING.md) and [code of conduct](https://github.com/privacy-scaling-explorations/maci/blob/dev/CODE_OF_CONDUCT.md).
- [x] I have read and understand MACI's [GitHub processes](https://github.com/privacy-scaling-explorations/maci/discussions/847).
- [x] I have read and understand MACI's [testing guide](https://maci.pse.dev/docs/testing).

### CODE REVIEW

File: cli/tests/e2e/e2e.test.ts [hunk 1]
```diff
@@ -68,20 +80,86 @@ describe("e2e tests", function test() {
...
+  const setVerifyingKeysArgs: SetVerifyingKeysArgs = {
+    quiet: true,
+    stateTreeDepth: STATE_TREE_DEPTH,
+    intStateTreeDepth: INT_STATE_TREE_DEPTH,
+    messageTreeDepth: MSG_TREE_DEPTH,
+    voteOptionTreeDepth: VOTE_OPTION_TREE_DEPTH,
+    messageBatchDepth: MSG_BATCH_DEPTH,
+    processMessagesZkeyPath: processMessageTestZkeyPath,
+    tallyVotesZkeyPath: tallyVotesTestZkeyPath,
+  };
```
Comment (0xmad) [inline cli/tests/e2e/e2e.test.ts:92]: Can we reuse these args for all the e2e tests? I noticed that almost the same args were using for e2e subsidy.

File: cli/ts/commands/airdrop.ts
Comment (0xmad) [inline cli/ts/commands/airdrop.ts:1]: I think we need to change jsdocs as well.

**Review by 0xmad — CHANGES_REQUESTED** [review]
@ctrlc03 thanks, just need to update the jsdoc for the functions and one question.

**Review by kittybest — APPROVED** [review]
lgtm

File: cli/tests/e2e/e2e.test.ts (hunk 1 above)
Comment (ctrlc03) [reply cli/tests/e2e/e2e.test.ts:92]: I moved everything but genProof args inside constants

File: cli/ts/commands/airdrop.ts
Comment (ctrlc03) [reply cli/ts/commands/airdrop.ts:1]: What you think I should put in the typedoc comments?

Comment (ctrlc03) [reply cli/ts/commands/airdrop.ts:1]: like: @param AirdropArgs - Arguments to the airdrop command?

Comment (0xmad) [reply cli/ts/commands/airdrop.ts:1]: Yes, and you can describe all the params for the interface https://stackoverflow.com/questions/6460604/how-to-describe-object-arguments-in-jsdoc

File: cli/ts/commands/airdrop.ts [hunk 2]
```diff
@@ -1,28 +1,21 @@
...
 /**
  * Utility that can be used to get
  * topup credits aidropped
  * to the coordinator
- * @param amount the amount of credits to airdrop
- * @param contractAddress the address of the ERC20 contract
- * @param pollId the id of the poll
- * @param maciAddress the address of the MACI contract
- * @param quiet whether to log the output
+ * @param AirdropArgs - The arguments for the airdrop command
```
Comment (0xmad) [inline cli/ts/commands/airdrop.ts:10]: I think it should be like this to navigate between types. Not sure if current approach will work with typedoc and navigation.

```suggestion
 * @param {AirdropArgs} - The arguments for the airdrop command
```

Comment (ctrlc03) [reply cli/ts/commands/airdrop.ts:10]: Screenshot 2024-01-18 at 17 46 55

Comment (ctrlc03) [reply cli/ts/commands/airdrop.ts:10]: after typedoc generates, it links good

Comment (ctrlc03) [reply cli/ts/commands/airdrop.ts:10]: is that ok?

**Review by 0xmad — APPROVED** [review]
@ctrlc03 thanks!

**Pull request merged**